rancher/rancher-compose/index.md

```diff
@@ -125,11 +137,13 @@ test2:
   - test-data
 ```
 
-### Cross-Stack Linking
+### Linking Services
+
+In Rancher, all services within an environment are DNS resolvable so linking services are not required unless you would like to use a specific alias for DNS resolution. 
 
-In Rancher, we allow linking of services across stacks, which is easy to represent in the `docker-compose.yml` file.
+> **Note:** We currently do not support linking sidekick services to the primary service or vice versa. Read more about how [Rancher's internal DNS works]({{site.baseurl}}/rancher/rancher-services/internal-dns-service/).
 
-Services in other stacks will be linked in `external_links`.
+For services in the same stack, any service is DNS resolvable by `service_name`. For services in a different stack, the service is DNS resolvable by `service_name.stack_name`. If you'd prefer to use a specific alias for DNS resolution, you would use `external_links` in the `docker-compose.yml`. 
```

Comment:
@deniseschannon 
> If you'd prefer to use a specific alias for DNS resolution, you would use `external_links` in the `docker-compose.yml`

using alias works not only cross stacks, but within the same stack as well. We should say something like "use links for service's alias within the same stack, and external-links when alias is created for the service in another stack"